data/mods/Magiclysm/monsters/Orcs.json

```
@@ -76,7 +83,8 @@
           "ranges": [ [ 12, 13, "DEFAULT" ] ],
           "no_ammo_sound": "grunting"
```

```suggestion
          "no_ammo_sound": "grunting."
```
The game won't add a period here or in `The orc shoots you with laser accuracy` so I guess we need to add them.